mortgage/E2E.ipynb

```diff
@@ -0,0 +1,673 @@
+    "        (\"net_sale_proceeds\", \"float64\"),\n",
+    "        (\"credit_enhancement_proceeds\", \"float64\"),\n",
+    "        (\"repurchase_make_whole_proceeds\", \"float64\"),\n",
+    "        (\"other_foreclosure_proceeds\", \"float64\"),\n",
+    "        (\"non_interest_bearing_upb\", \"float64\"),\n",
+    "        (\"principal_forgiveness_upb\", \"float64\"),\n",
+    "        (\"repurchase_make_whole_proceeds_flag\", \"category\"),\n",
+    "        (\"foreclosure_principal_write_off_amount\", \"float64\"),\n",
+    "        (\"servicing_activity_indicator\", \"category\")\n",
+    "    ])\n",
+    "\n",
+    "    print(performance_path)\n",
+    "    \n",
+    "    return cudf.read_csv(performance_path, names=cols, delimiter='|', dtype=list(dtypes.values()), skiprows=1)\n",
+    "\n",
+    "def gpu_load_acquisition_csv(acquisition_path, **kwargs):\n",
+    "    \"\"\" Loads acquisition data\n",
+    "\n",
+    "    Returns\n",
+    "    -------\n",
+    "    GPU DataFrame\n",
+    "    \"\"\"\n",
+    "    \n",
+    "    cols = [\n",
+    "        'loan_id', 'orig_channel', 'seller_name', 'orig_interest_rate', 'orig_upb', 'orig_loan_term', \n",
+    "        'orig_date', 'first_pay_date', 'orig_ltv', 'orig_cltv', 'num_borrowers', 'dti', 'borrower_credit_score', \n",
+    "        'first_home_buyer', 'loan_purpose', 'property_type', 'num_units', 'occupancy_status', 'property_state',\n",
+    "        'zip', 'mortgage_insurance_percent', 'product_type', 'coborrow_credit_score', 'mortgage_insurance_type', \n",
+    "        'relocation_mortgage_indicator'\n",
+    "    ]\n",
+    "    \n",
+    "    dtypes = OrderedDict([\n",
+    "        (\"loan_id\", \"int64\"),\n",
+    "        (\"orig_channel\", \"category\"),\n",
+    "        (\"seller_name\", \"category\"),\n",
+    "        (\"orig_interest_rate\", \"float64\"),\n",
+    "        (\"orig_upb\", \"int64\"),\n",
+    "        (\"orig_loan_term\", \"int64\"),\n",
+    "        (\"orig_date\", \"date\"),\n",
+    "        (\"first_pay_date\", \"date\"),\n",
+    "        (\"orig_ltv\", \"float64\"),\n",
+    "        (\"orig_cltv\", \"float64\"),\n",
+    "        (\"num_borrowers\", \"float64\"),\n",
+    "        (\"dti\", \"float64\"),\n",
+    "        (\"borrower_credit_score\", \"float64\"),\n",
+    "        (\"first_home_buyer\", \"category\"),\n",
+    "        (\"loan_purpose\", \"category\"),\n",
+    "        (\"property_type\", \"category\"),\n",
+    "        (\"num_units\", \"int64\"),\n",
+    "        (\"occupancy_status\", \"category\"),\n",
+    "        (\"property_state\", \"category\"),\n",
+    "        (\"zip\", \"int64\"),\n",
+    "        (\"mortgage_insurance_percent\", \"float64\"),\n",
+    "        (\"product_type\", \"category\"),\n",
+    "        (\"coborrow_credit_score\", \"float64\"),\n",
+    "        (\"mortgage_insurance_type\", \"float64\"),\n",
+    "        (\"relocation_mortgage_indicator\", \"category\")\n",
+    "    ])\n",
```

Comment:
Are you missing this line? `("quarter", "int64")`

Reply:
This is missing. Thanks for the catch.